{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/base/define.css

```diff
@@ -7,6 +7,26 @@
 | Global resets to make life easier
 |
 */
+a,
+area,
+button,
+[role='button'],
+input:not([type='range']),
+label,
+select,
+summary,
+textarea {
+  /* stylelint-disable property-no-vendor-prefix */
+  -ms-touch-action: manipulation;
+  touch-action: manipulation;
+}
```

Comment:
Is this related to the tabbing fix? I am not sure how.

Reply:
(It's fine if it's not - I'm just curious if it is)

Reply:
It's not but i added it as it was semi along the same lines. It's for focus/active on iOS

Reply:
Seems related enough to me :)